### PR TITLE
Update for Erigon RPC

### PR DESCRIPTION
**Pull request by zjg555543**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,23 +2,23 @@ Welcome to X Layer
...
+| Mainnet   | [v2.61.0](https://github.com/0xPolygonHermez/cdk-erigon/releases/tag/v2.61.0)    |
+| Testnet   | [v2.61.0](https://github.com/0xPolygonHermez/cdk-erigon/releases/tag/v2.61.0)    |
 
+## zkEVM RPC Version [Will be deprecated]
 
-## Erigon RPC Versions
-Please refer to [cdk-erigon](https://github.com/0xPolygonHermez/cdk-erigon)
+| Network   | Node      | Prover/Executor   | 
+| ---       | ---       | ---               | 
+| Mainnet   | [v0.4.0](https://github.com/okx/xlayer-node/releases/tag/v0.4.0)    |  [v0.3.2](https://github.com/okx/xlayer-prover/releases/tag/v0.3.2)    |
```
Comment (Vui-Chee) [inline README.md:16]: Our xlayer-node also has [v0.4.1](https://github.com/okx/xlayer-node/releases/tag/v0.4.1), are we planning to deploy that?

File: README.md [hunk 2]
```diff
@@ -2,23 +2,23 @@ Welcome to X Layer
...
+| Network   | Node      | Prover/Executor   | 
+| ---       | ---       | ---               | 
+| Mainnet   | [v0.4.0](https://github.com/okx/xlayer-node/releases/tag/v0.4.0)    |  [v0.3.2](https://github.com/okx/xlayer-prover/releases/tag/v0.3.2)    |   
+| Testnet   | [v0.4.0](https://github.com/okx/xlayer-node/releases/tag/v0.4.0)    |  [v0.3.2](https://github.com/okx/xlayer-prover/releases/tag/v0.3.2)    |  
 
 ## Document Navigation
 
 | Testnet      |  Mainnet  |
 | ---       | ---               |
-|   [Setup/Upgrade zkEVM RPC Node](./testnet/setup-zknode-rpc.md)      |   [Setup/Upgrade zkEVM RPC Node](./mainnet/setup-zknode-rpc.md)   |
```
Comment (Vui-Chee) [inline README.md:21]: This is removed, so is the document in this repo also removed? Or has documentation moved to new location?

File: mainnet/setup-erigon-rpc.md [hunk 3]
```diff
@@ -12,7 +12,7 @@ cdk-erigon is a fork of Erigon, optimized for syncing with the XLayer network.
 ``` bash
 git clone https://github.com/0xPolygonHermez/cdk-erigon.git
 cd cdk-erigon
-git checkout v1.2.15.3 #[checkout the latest release version]
+git checkout v2.61.0 #[checkout the latest release version]
```
Comment (Vui-Chee) [inline mainnet/setup-erigon-rpc.md:15]: For the `git clone`, should we be deploying our forked version, not the upstream?

File: README.md [hunk 4]
```diff
@@ -2,23 +2,23 @@ Welcome to X Layer
...
+## Erigon RPC Version [Recommend]
 
-| Network   | Node      | Prover/Executor   |
-| ---       | ---       | ---               |
-| Mainnet   | [v0.4.0](https://github.com/okx/xlayer-node/releases/tag/v0.4.0)    |  [v0.3.2](https://github.com/okx/xlayer-prover/releases/tag/v0.3.2)    |
-| Testnet   | [v0.4.0](https://github.com/okx/xlayer-node/releases/tag/v0.4.0)    |  [v0.3.2](https://github.com/okx/xlayer-prover/releases/tag/v0.3.2)    |
+| Network   |             Erigon       |
+| ---       | ---                      |
+| Mainnet   | [v2.61.0](https://github.com/0xPolygonHermez/cdk-erigon/releases/tag/v2.61.0)    |
+| Testnet   | [v2.61.0](https://github.com/0xPolygonHermez/cdk-erigon/releases/tag/v2.61.0)    |
```
Comment (Vui-Chee) [inline README.md:10]: Should we be deploying our forked repo?

Comment (Vui-Chee) [reply README.md:10]: Adding to that, our fork repo tags stops at [release/v0.4.0](https://github.com/okx/xlayer-erigon/releases/tag/release%2Fv0.4.0). Apparently, the upstream tags are not synced with our fork.

File: README.md (hunk 2 above)
Comment (zjg555543) [reply README.md:21]: Remove it after upgrade

**Review by giskook — APPROVED** [review]

**Pull request merged**